org.prismlauncher.PrismLauncher.yml

```diff
@@ -0,0 +1,90 @@
+app-id: org.prismlauncher.PrismLauncher
+runtime: org.kde.Platform
+runtime-version: "5.15-21.08"
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+  - org.freedesktop.Sdk.Extension.openjdk8
+add-extensions:
+  com.valvesoftware.Steam.Utility.gamescope:
+    version: stable
+    add-ld-path: lib
+    no-autodownload: true
+    directory: utils/gamescope
+
+command: prismlauncher
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
does it support wayland?

Reply:
The launcher itself does support Wayland, yes, as it's using Qt.
The game can be made to work with Wayland as well, but that doesn't work right now without further modifications.